cli/tests/pull.bats

```diff
@@ -554,6 +554,47 @@ function add_incompatible_package() {
 
 # ---------------------------------------------------------------------------- #
 
+# bats test_tags=pull:copy:new
+@test "'pull --copy' creates path environment" {
+  make_dummy_env "owner" "name"
+
+  # dummy environment has no packages to resolve
+  export _FLOX_USE_CATALOG_MOCK="$GENERATED_DATA/empty.json"
```

Comment:
nit: that's already set in setup

Reply:
i see, yeah the test i copied to start off of had this in it.
Ill just remove it from that one as well